### DISCUSSION TITLE
Is it possible to capture an event to multiple notes with conditions?

### DISCUSSION THREAD

**Discussion by ooker777** [chhoumann/quickadd#370] [Q&A]
I want to build a vault for logging events, similar to [this example QuickAdd capture](https://quickadd.obsidian.guide/docs/Examples/Capture_AddJournalEntry). Each event will append an entry in the form `HH:mm [[activity name]] [with:: [[person]]]`. For example:
```
- 08:31 [[Swimming]] 
- 10:45 [[Eat delicious food]] [with:: [[Alice]]] 
- 15:19 [[Meeting]] [with:: [[Alice]], [[Bob]]] 
```
If the field `with` exists, then the events are also capture in the person log. So `[[Alice]]` will have
```
- 10:45 [[Eat delicious food]] [with:: [[Alice]]] 
- 15:19 [[Meeting]] [with:: [[Alice]], [[Bob]]] 
```
and `[[Bob]]` will have
```
- 15:19 [[Meeting]] [with:: [[Alice]], [[Bob]]] 
```

Is that possible?

<sub>Also asked in [Obsidian Forum](https://forum.obsidian.md/t/is-it-possible-to-build-a-log-from-the-daily-notes-with-dataview/41109)</sub>

**Comment by chhoumann** [comment]
You can capture to multiple notes with macros. Variables populated in a macro step will be accessible / shared in subsequent steps.

For this specific use case, you'd probably want to create a script that figures out which people are mentioned.
Once you have those files, you can use the QuickAdd API to use a capture on each of those, given the data you have (e.g. event & time).
